Mathlib/Order/Filter/Germ.lean

```diff
@@ -361,26 +361,21 @@ theorem liftRel_const_iff [NeBot l] {r : β → γ → Prop} {x : β} {y : γ} :
   @eventually_const _ _ _ (r x y)
 #align filter.germ.lift_rel_const_iff Filter.Germ.liftRel_const_iff
 
-instance inhabited [Inhabited β] : Inhabited (Germ l β) :=
-  ⟨↑(default : β)⟩
+instance instInhabited [Inhabited β] : Inhabited (Germ l β) := ⟨↑(default : β)⟩
 
 section Monoid
 
 variable {M : Type*} {G : Type*}
 
-@[to_additive]
-instance mul [Mul M] : Mul (Germ l M) :=
-  ⟨map₂ (· * ·)⟩
+@[to_additive] instance instMul [Mul M] : Mul (Germ l M) := ⟨map₂ (· * ·)⟩
```

Comment:
Why is `to_additive` on the same line now? I have no idea what the standardization of this is supposed to be.

Reply:
If it fits on a line, you can put it on a line. This is how it always was